projects/v3/src/app/pages/activity-desktop/activity-desktop.page.ts

```diff
@@ -76,6 +78,7 @@ export class ActivityDesktopPage implements OnInit {
     } else {
       this.loading = false;
     }
+    this.savingText = 'Last saved ' + this.utils.getFormatedCurrentTime();
```

Comment:
the function will return `if (!event.assessment.inProgress)` so this should come before that